components/logins/android/build.gradle

```diff
@@ -92,5 +107,6 @@ afterEvaluate {
 }
 
 apply from: "$rootDir/publish.gradle"
+apply from: "https://github.com/mozilla-mobile/android-components/raw/v${android_components_version}/components/service/glean/scripts/sdk_generator.gradle"
```

Comment:
```suggestion
// Generate markdown docs for the collected metrics.
ext.gleanGenerateMarkdownDocs = true
ext.gleanDocsDirectory = "$rootDir/docs/collected-metrics/logins"
apply from: "https://github.com/mozilla-mobile/android-components/raw/v${android_components_version}/components/service/glean/scripts/sdk_generator.gradle"
```

Reply:
Oh sorry, I forgot to mention this, apologies. You can get automatically generated human-readable docs from Glean. By adding the above two lines it will generate docs (and keep them up to date if `metrics.yaml` or `pings.yaml` ever change!) that you can keep in-tree in this directory: `docs/collected-metrics/logins` (of course you can change the directory).

Reply:
Is this something data stewards should responsible for observing?

Reply:
Added, along with a changelog entry calling it out as a breaking change. Thanks!

Reply:
> 
> 
> Is this something data stewards should responsible for observing?

OT: It would be great! What if we organise some meeting between Data Stewards & Glean team?